docs/tenancy/1.x/tutorial-non-tenant.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Non-Tenant Site Setup
+icon: fal fa-gears
+excerpt: >
+    How to setup goabal and admin sites in the same application
+tags:
+    - tenancy
+    - tutorial
+    - non-tenant
+---
+
+## Introduction
+When building a SaaS there tends to be a need for a "sub-site" such as an admin panel, or a public facing website describing your SaaS.
+
+This tutorial will go though the basics of setting these "sub-sites" within the same codebase without the need for creating a tenant do display these sites.
+
+## Requirements
+This tutorial will assume that you have followed the [basic setup tutorial](tutorial-basic-setup).
+
+## Routes
+The first thing we want to do is configure the [Routes Affect](affacts-routes).
+
+First start by creating a new file for all tenant specific routes.
+
+### Tenant Route File
+For this example it will be stored at `routes\tenant.php`
+```php
+use Illuminate\Support\Facades\Route;
+
+/*
+|--------------------------------------------------------------------------
+| Tenant Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your tenant application.
+| They are not loaded by default, and will be loaded thought the Routes Affect.
+|
+*/
+
+Route::get('/', function () {
+    //This is the "root" route for all tenants
```

Comment:
I'd never use a callable in an example for routes. Callables cause routes files to be no longer cacheable. So.. either just remove the non-working example or provide a controlller from the get-go.